tasks/populate-release-notes-images/populate-release-notes-images.yaml

```diff
@@ -64,6 +65,15 @@ spec:
             repository=$(echo ${containerImage} | cut -d '/' -f 2- | cut -d '@' -f 1)
             # purl should be pkg:oci/foo@sha256:abcde?repository_url=registry.redhat.io/foo/bar
             purl="pkg:oci/$(echo $repository | cut -d '/' -f 1)@sha256:${sha}?repository_url=${deliveryRepo}"
+            # Construct CVE json
+            CVEsJson='{"cves":{"fixed":{}}}'
+            for cve in $(jq -c '.releaseNotes.cves[] | select(.component=="'$name'")' ${DATA_FILE}) ; do
+                cveJson=$(jq -n \
+                    --arg id $(jq -r '.key' <<< $cve) \
+                    --argjson packages $(jq -c '.packages' <<< $cve) \
+                    '{($id): {"components": $packages}}')
+                CVEsJson=$(jq --argjson cve "$cveJson" '.cves.fixed += $cve' <<< $CVEsJson)
+            done
             # Add one entry per arch (amd64 for example)
             get-image-architectures "${image}" | while IFS= read -r arch_json;
```

Comment:
you might need a `jq -c`  on `get-image-architectures` output to keeping using the while loop as it now outputs a formatted json. The option would be change the script to output the inline json instead. 

I just pushed a fix to create-pyxis-image that broke after the last change https://github.com/redhat-appstudio/release-service-catalog/pull/398/files#diff-a98bb6e022384823267a8899f22bc893823067c9cbbc366256a763e0d9249536L162

Reply:
base image of the task (and tests) updated for the compact json fix